### DISCUSSION TITLE
Is markdoc suitable for a small SaaS API documentation?

### DISCUSSION THREAD

**Discussion by MJomaa** [markdoc/markdoc#326] [Q&A] [answered]
About 40 endpoints need to be documented.

With markdoc I can really influence the style and integrate it in our nextjs site unlike swagger, slate or redoc.

Currently evaluating if one should use react-markdown + graymatter, markdoc, gitbook or something else.

**Comment by mfix-stripe** [comment, marked as answer]
Hey @MJomaa — we are currently writing our own API documentation at Stripe in Markdoc, which has far more than 40 endpoints. Whether or not you should use Markdoc probably comes down to specific tradeoffs that you want to make between authoring experience, level of extensibility/customizability, etc., which will be difficult for me to assess from the outside. 

All that said, Markdoc is a low-level library for documentation, so it can _definitely_ account for your needs. Whether it accounts for them with less effort than other solutions depends on a number of factors.
